continuous_integration/scripts/install.sh

```diff
@@ -7,11 +7,14 @@ set -xe
 # python -m pip install --no-deps cityhash
 
 if [[ ${UPSTREAM_DEV} ]]; then
-    conda uninstall --force numpy pandas
-    python -m pip install --no-deps --pre \
-        -i https://pypi.anaconda.org/scipy-wheels-nightly/simple \
-        numpy
-    python -m pip install --pre pandas==1.1.0rc0
+    # conda uninstall --force numpy pandas
+    # python -m pip install --no-deps --pre \
+    #     -i https://pypi.anaconda.org/scipy-wheels-nightly/simple \
+    #     numpy
+    # python -m pip install --pre pandas==1.1.0rc0
```

Comment:
Apologies for not catching this earlier, but I think we want to keep this for testing nightly versions of numpy and pandas

Though I just noticed we're pinning `pandas==1.1.0rc0` which might be an oversight. Instead I would have expected us to install both numpy and pandas from https://pypi.anaconda.org/scipy-wheels-nightly/simple

Reply:
Yes, we certainly want to keep that, but the thing is that they were currently failing a lot, and thus not very useful (hard to see what errors still need to be fixed for a given dependency). The pandas ones are being fixed in https://github.com/dask/dask/pull/6996 (and the pandas nightly install is added back there), and the numpy ones in https://github.com/dask/dask/pull/6896 I think

Reply:
Great! Thanks for clarifying